strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/maven/MavenSearchControllerTest.java

```diff
@@ -68,7 +69,7 @@ public void testDbSearches(@MavenRepository(storageId = STORAGE_SC_TEST,
             throws Exception
     {
         testSearches("groupId=org.carlspring.strongbox.searches;artifactId=test-project;",
-                     OrientDbSearchProvider.ALIAS);
+                     "OrientDbSearchProvider.ALIAS");
```

Comment:
This makes no sense. I'm assuming this test case (method) is now obsolete. Could you please remove it and have another look at the whole class? If it's obsolete, just remove the class.

Reply:
this test disabled 
just leaved it here as a reminder how it was before

Reply:
Okay.
Add a comment in the code, please.

Reply:
done

Reply:
Thanks!